src/api/Config.js

```diff
@@ -155,6 +155,125 @@ const config = [
       },
     ],
   },
+  {
+    name: 'tractionv2',
```

Comment:
What is this for? We do not have a traction api v2 in the service.

Reply:
That's something on trial, that I forgot to remove 🤦‍♀️